app-laptop/msi-keyboard/metadata.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>zohran.londais@gmail.com</email>
+	</maintainer>
+	<longdescription lang="en">
```

Comment:
You'll need to add someone with @gentoo.org email to commit for you. It's usually the proxy-maint project, 
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Proxied_maintainer_in_metadata.xml

Check examples from other proxy-maintained ebuilds.

Reply:
Can i add anybody, or i need to ask somebody if he is agree to commit for me ?

Reply:
Yes, you need your committers consent. If you dont have anyone in mind, add the proxy-maint project (see the link I gave you above)